src/main/java/org/jitsi/videobridge/octo/OctoRelay.java

```diff
@@ -41,16 +44,20 @@
 
     /**
      * The Octo relay ID which will be advertised by jitsi-videobridge.
+     * Other bridges can use this ID in order to discover the socket address
+     * that this bridge is accessible on. With the current implementation the
+     * ID just encodes a pre-configured IP address and port,
+     * e.g. "10.0.0.1:20000"
```

Comment:
thanks!  so, other bridges must be able to use this id directly (without any other mapping) to know how to reach this bridge?  if so, it will have to be an address, won't it? does it make sense to define this as a String as opposed to something that more strictly models an address and would be serialized? (and i'm assuming it would be something like InetAddress here)